### ISSUE TITLE
EPIC 221823709 (Colin)

### ISSUE THREAD

**Issue by George-Hookway** [George-Hookway/PBjammy#14]
![221823709 - 14 - MS - Echelle](https://github.com/user-attachments/assets/67cfce58-2c04-4d0f-9ce3-32c10dead991)
![221823709 - 14 - MS - Mode ID](https://github.com/user-attachments/assets/c1c71867-5994-4ae2-9540-88d61a9d1efc)
![221823709 - 14 - MS - Peakbagged Echelle](https://github.com/user-attachments/assets/4df73844-a722-4acc-8011-62239c176fa0)
![221823709 - 14 - MS - Peakbagging](https://github.com/user-attachments/assets/63bb69f0-7848-43e9-b81f-a62da4ea47d9)

@nielsenmb @grd349

**Comment by George-Hookway** [comment]
The Keystone numax might be a little bit too high, but I think all the identifiable modes have been identified anyway.

**Comment by nielsenmb** [comment]
Yup, looks good. The constant linewidth approximation in the mode ID stage looks to be faltering a little bit, but it doesn't seem to be impacting things at all.
